SpicesUpdater@claudiux/files/SpicesUpdater@claudiux/assets/spawn.js

```diff
@@ -0,0 +1,33 @@
+const Gio   = imports.gi.Gio;
+const GLib  = imports.gi.GLib;
+const Lang  = imports.lang;
+
+var SpawnReader = function () { };
+
+SpawnReader.prototype.spawn = function (path, command, func) {
```

Comment:
Do we need to re-invent the wheel here? Cinnamon has a lot of utilities internally that makes this more compact. If you just need to asynchronously grab stdout and be notified of success or error, a function like [this one](https://github.com/linuxmint/cinnamon-spices-applets/blob/07449f8bc6e32218be2fa9c0f532344df4efa27b/IcingWindowSaver%40json/files/IcingWindowSaver%40json/3.6/applet.js#L20), or the one in this [Cinnamon PR](https://github.com/linuxmint/Cinnamon/pull/8130) would work well, and makes handling async callbacks easier.

Reply:
I removed entirely the folder `assets`, and the file `egSpawn.js`.
Now, `tail` and `zenity` are doing the job, with a few lines of code.

Thanks for all, @jaszhix !